### PR TITLE
Add notes about ZenHub pielines to CONTRIBUTING.

### PR DESCRIPTION
**Pull request by anorth**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -157,6 +158,20 @@ To pick up an issue:
 
 For continued adventures, search for issues with the label [E-help-wanted](https://github.com/filecoin-project/go-filecoin/issues?q=is%3Aopen+is%3Aissue+label%3AE-help-wanted). These are slightly thornier problems that are also reasonably well-prepared.
 
+### Pipelines (ZenHub)
+We use ZenHub pipelines to track the flow of work on open issues.
```
Comment (mishmosh) [inline CONTRIBUTING.md]: We should add a link to Zenhub: https://app.zenhub.com/workspaces/filecoin-5ab0036a12f8e82ae4ed60f0/board?repos=113219518,158620468&showPipelineDescriptions=false

Comment (mishmosh) [reply CONTRIBUTING.md]: I think it's worth calling out that general contributors do not need, but committers & maintainers are expected to keep pipelines info up to date.

Comment (anorth) [reply CONTRIBUTING.md]: The ZenHub link is a few lines above here (but not including the private repo).

I added a sentence about expectations as you suggest.

**Comment by laser** [comment]
Pinging @mishmosh and @travisperson for review

**Comment by anorth** [comment]
Ping @travisperson

**Pull request merged**